diff --git a/node_modules/expo-router/_ctx.android.js b/node_modules/expo-router/_ctx.android.js
index 1afb1f8..855c64c 100644
--- a/node_modules/expo-router/_ctx.android.js
+++ b/node_modules/expo-router/_ctx.android.js
@@ -1,6 +1,6 @@
 export const ctx = require.context(
-  process.env.EXPO_ROUTER_APP_ROOT,
+  '../../src/app',
   true,
   /^(?:\.\/)(?!(?:(?:(?:.*\+api)|(?:\+html)|(?:\+middleware)))\.[tj]sx?$).*(?:\.ios|\.web)?\.[tj]sx?$/,
-  process.env.EXPO_ROUTER_IMPORT_MODE
+  'sync'
 );
diff --git a/node_modules/expo-router/_ctx.ios.js b/node_modules/expo-router/_ctx.ios.js
index b30ee6c..4434f5d 100644
--- a/node_modules/expo-router/_ctx.ios.js
+++ b/node_modules/expo-router/_ctx.ios.js
@@ -1,6 +1,6 @@
 export const ctx = require.context(
-  process.env.EXPO_ROUTER_APP_ROOT,
+  '../../src/app',
   true,
   /^(?:\.\/)(?!(?:(?:(?:.*\+api)|(?:\+html)|(?:\+middleware)))\.[tj]sx?$).*(?:\.android|\.web)?\.[tj]sx?$/,
-  process.env.EXPO_ROUTER_IMPORT_MODE
+  'sync'
 );
diff --git a/node_modules/expo-router/_ctx.js b/node_modules/expo-router/_ctx.js
index 0979e39..d2355c7 100644
--- a/node_modules/expo-router/_ctx.js
+++ b/node_modules/expo-router/_ctx.js
@@ -1,5 +1,5 @@
 export const ctx = require.context(
-  process.env.EXPO_ROUTER_APP_ROOT,
+  '../../src/app',
   true,
   // Ignore root `./+html.js` and API route files `./generate+api.tsx`.
   /^(?:\.\/)(?!(?:(?:(?:.*\+api)|(?:\+html)))\.[tj]sx?$).*\.[tj]sx?$/
diff --git a/node_modules/expo-router/_ctx.web.js b/node_modules/expo-router/_ctx.web.js
index 8061699..16b7993 100644
--- a/node_modules/expo-router/_ctx.web.js
+++ b/node_modules/expo-router/_ctx.web.js
@@ -1,6 +1,6 @@
 export const ctx = require.context(
-  process.env.EXPO_ROUTER_APP_ROOT,
+  '../../src/app',
   true,
   /^(?:\.\/)(?!(?:(?:(?:.*\+api)|(?:\+middleware)|(?:\+(html|native-intent))))\.[tj]sx?$).*(?:\.android|\.ios|\.native)?\.[tj]sx?$/,
-  process.env.EXPO_ROUTER_IMPORT_MODE
+  'sync'
 );
